libkbfs/prefetcher.go

```diff
@@ -296,8 +296,8 @@ top:
 // calculatePriority returns either a base priority for an unsynced TLF or a
 // high priority for a synced TLF.
 func (p *blockPrefetcher) calculatePriority(
-	basePriority int, isDeepSync bool) int {
-	if isDeepSync {
+	basePriority int, action BlockRequestAction) int {
+	if action.DeepSync() {
 		return defaultOnDemandRequestPriority - 1
```

Comment:
I think this should probably just be `basePriority` now, where each level sets the priority of triggered requests to 1 less than the parent block. The root block of a deep sync will have a much lower base priority than working set triggered prefetches, which will just start at `defaultOnDemandRequestPriority`.